### ISSUE TITLE
coverimage doesn't work with relative URL unless leading `/` added

### ISSUE THREAD

**Issue by zeigerpuppy** [kakawait/hugo-tranquilpeak-theme#236]
### Configuration

 - **Hugo version**: 0.31
 - **Tranquilpeak version**: 0.4.3-BETA

### Actual behavior
In a post header, adding a coverImage works with a URL but not a relative link, unless a leading `/` is added.

### Expected behavior
A relative link should work without a leading slash (it does for `thumbnailImage`).

this fails for an image in `static/images/posts/`
```
coverImage: images/posts/hugo_cover.jpg
```

this works
```
coverImage: /images/posts/hugo_cover.jpg
```

**Comment by kakawait** [comment]
Thank for reporting i will checkout how to fix it.

**Comment by Titan-C** [comment]
Currently in master is not great. I had for coverImage an Image from another host. Thus 
```
coverImage: https://some.host/_images/img.jpg
```
The current solution prefixes an "/" making then the address invalid `/https.//somehost`. Maybe this is a bad practice and I should use `//` for external links but that as long as everybody is using https.

**Comment by kakawait** [comment]
@Titan-C not is a bug a reopen issue i will fix it on minor release 0.4.4-BETA
